frontend/public/components/machine-config.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as React from 'react';
+
+import { MachineConfigKind, referenceForModel } from '../module/k8s';
+import { MachineConfigModel } from '../models';
+import { fromNow } from './utils/datetime';
+import {
+  ColHead,
+  DetailsPage,
+  List,
+  ListHeader,
+  ListPage,
+} from './factory';
+import {
+  Kebab,
+  navFactory,
+  ResourceKebab,
+  ResourceLink,
+  ResourceSummary,
+  SectionHeading,
+} from './utils';
+
+
+export const machineConfigReference = referenceForModel(MachineConfigModel);
+export const machineConfigMenuActions = Kebab.factory.common;
+
+export const MachineConfigSummary: React.SFC<MachineConfigSummaryProps> = ({obj}) => (
+  <ResourceSummary resource={obj} showPodSelector={false} showNodeSelector={false}>
+    <dt>OS Image URL</dt>
+    <dd>{obj.spec.osImageURL || '-'}</dd>
+  </ResourceSummary>
+);
+
+const MachineConfigDetails: React.SFC<MachineConfigDetailsProps> = ({obj}) => (
+  <div className="co-m-pane__body">
+    <SectionHeading text="Machine Config Overview" />
+    <div className="row">
+      <div className="col-sm-6">
```

Comment:
Why col-sm-6 if we only have one column?

Reply:
Copy paste oversight